app/code/core/Mage/Adminhtml/Block/System/Config/Form.php

```diff
@@ -437,7 +441,7 @@ public function initFields($fieldset, $group, $section, $fieldPrefix = '', $labe
                                 $optionArray[] = ['label' => $label, 'value' => $value];
                             }
                         }
-                    } else {
+                    } elseif (method_exists($sourceModel, 'toOptionArray')) {
                         $optionArray = $sourceModel->toOptionArray($fieldType == 'multiselect');
                     }
                     $field->setValues($optionArray);
```

Comment:
`$optionArray` can be undefined. Move line 435?

![image](https://user-images.githubusercontent.com/1106470/225484637-d3b72194-104b-4e22-8d7c-0fe835f859ee.png)

Reply:
moved the declaration, is it enought?

Reply:
done :-)